lib/src/common/variables.ts

```diff
@@ -390,6 +395,9 @@ export const componentTokens = {
     socialLinksSize: "24px",
     socialLinksGutter: "16px",
     socialLinksColor: CoreTokens.color_white,
+    reducedHeight: "40px",
```

Comment:
We should not have `reducedHeight` token, since the height of the reduced footer should not be changed.